### DISCUSSION TITLE
How can mirror root storage's contents(images and operators) in mirror-registry to anther mirror-registry

### DISCUSSION THREAD

**Discussion by biruand1016** [okd-project/okd#1363] [Q&A] [answered]
Hi, team
We installed local mirror registry by using Red Hat Mirror Registry. But we have to construct mirror registry on another computer and we are going to use root storage(/opt/mirror/data) in origin mirror registry, that is,
We copy root storage to USB and take the usb to another computer.
We copy the root storage in the computer, and install new mirror registry,
We connect copied root storage to the new mirror registry. 
Is possible this?
How can use origin root storage  in the new mirror-registry?

**Comment by vrutkovs** [comment, marked as answer]
See https://docs.okd.io/latest/installing/disconnected_install/installing-mirroring-disconnected.html#mirroring-image-set-full

**Comment by biruand1016** [comment]
thank you for your help
